src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/android-agent-crash-reporting.mdx

```diff
@@ -61,6 +61,17 @@ To enable compressed map uploads, add the following to your app's `newrelic.prop
 com.newrelic.compressed_uploads=true
 ```
 
+#### Configure mapping upload host
+
+Add the following to your app's `newrelic.properties` file to send maps on a redirected server:
+
+```properties
+com.newrelic.mapping_upload_host=<redirected.domain.com>
```

Comment:
If you had a problem here with a specific format of the redirected domain name (_ending with a /, iirc), call it out here with a concrete example. It would be better to harden the code that reads this property, so it wouldn't matter if a user ended the domain URI with a slash, or started with an unsupported scheme, (`http://`), etc.